chpl/chpl-service/src/main/java/gov/healthit/chpl/changerequest/domain/service/ChangeRequestWebsiteService.java

```diff
@@ -0,0 +1,207 @@
+package gov.healthit.chpl.changerequest.domain.service;
+
+import java.text.DateFormat;
+import java.util.HashMap;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.mail.MessagingException;
+
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.env.Environment;
+import org.springframework.stereotype.Component;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+
+import gov.healthit.chpl.changerequest.dao.ChangeRequestDAO;
+import gov.healthit.chpl.changerequest.dao.ChangeRequestWebsiteDAO;
+import gov.healthit.chpl.changerequest.domain.ChangeRequest;
+import gov.healthit.chpl.changerequest.domain.ChangeRequestWebsite;
+import gov.healthit.chpl.dao.DeveloperDAO;
+import gov.healthit.chpl.dao.UserDeveloperMapDAO;
+import gov.healthit.chpl.domain.Developer;
+import gov.healthit.chpl.domain.activity.ActivityConcept;
+import gov.healthit.chpl.dto.DeveloperDTO;
+import gov.healthit.chpl.dto.auth.UserDTO;
+import gov.healthit.chpl.exception.EntityCreationException;
+import gov.healthit.chpl.exception.EntityRetrievalException;
+import gov.healthit.chpl.exception.MissingReasonException;
+import gov.healthit.chpl.exception.ValidationException;
+import gov.healthit.chpl.manager.ActivityManager;
+import gov.healthit.chpl.manager.DeveloperManager;
+import gov.healthit.chpl.util.EmailBuilder;
+
+@Component
+public class ChangeRequestWebsiteService implements ChangeRequestDetailsService<ChangeRequestWebsite> {
+
+    private ChangeRequestDAO crDAO;
+    private ChangeRequestWebsiteDAO crWebsiteDAO;
+    private DeveloperDAO developerDAO;
+    private DeveloperManager developerManager;
+    private UserDeveloperMapDAO userDeveloperMapDAO;
+    private ActivityManager activityManager;
+    private Environment env;
```

Comment:
Consider removing the Environment autowire and replacing env.getProperty(s) with @Value like the other fields.